src/lib/server/blog/posts.ts

```diff
@@ -37,6 +39,12 @@ export async function fetchBlogPosts(options?: BlogFetchOptions): Promise<BlogOu
     });
 
     allPosts = cacheBlogPosts(await response.json());
+
+    // Iterate over all blog posts, applying any post processing steps defined
+    // in _postProcessHTML()
+    for (const post of allPosts) {
+      post.html = _postProcessHTML(post.html);
```

Comment:
Consider calling `postProcessHTML` in `cacheBlogPosts` on [line 107](https://github.com/EthanThatOneKid/acmcsuf.com/pull/831/files#diff-f01d7cf7b6cdb860c79a81684a3b6a0c20a4857de2b5d0b5e7eedca94fe07c6aR107).